build/conda/installer/post_install.sh

```diff
@@ -1,39 +1,89 @@
 #!/bin/bash
 
-# Full path to the Python executable inside the constructed environment
+# Setup environment.
+export PATH="$PREFIX/bin:$PATH"
 PYTHON_EXEC="$PREFIX/bin/python"
+IPYTHON_EXEC="$PREFIX/bin/ipython"
+REQUIREMENTS_FILE="$PREFIX/requirements.txt"
 LOG_FILE="$PREFIX/post_install_log.txt"
+CWDIR=$(dirname "$PREFIX")
 
-# Function to add timestamp
+# Function to add timestamp.
 log_with_timestamp() {
     echo "$(date '+%Y-%m-%d_%H:%M:%S') $1" >>"$LOG_FILE"
 }
 
-# Use the specific Python that comes bundled with the installer
-if "$PYTHON_EXEC" -m pip install -U "openbb[all]" openbb-cli openbb-platform-pro-backend >>"$LOG_FILE" 2>&1; then
+# Update pip and setuptools.
+"$PYTHON_EXEC" -m pip install -U pip >>"$LOG_FILE" 2>&1
+
+"$PYTHON_EXEC" -m pip install -U setuptools >>"$LOG_FILE" 2>&1
+
+"$PYTHON_EXEC" -m pip install poetry >>"$LOG_FILE" 2>&1
+
+# Install OpenBB packages.
+if "$PYTHON_EXEC" -m pip install -U -r "$REQUIREMENTS_FILE" >>"$LOG_FILE" 2>&1; then
     log_with_timestamp "pip install completed successfully."
 else
     log_with_timestamp "Error during post-installation: pip install failed."
     exit 1
 fi
 
-# Build OpenBB's python interface
+# Build static assets.
```

Comment:
We call it python interface in the windows script and static assets here. I don't have any preference, but would ask for consistency